source/stubs/returning-teams-stub.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+Returning Teams
+===============
+
+This section serves as a launch point for various popular articles that a veteran team may want. It is advised that **all** teams read through the :doc:`changelog </docs/yearly-overview/yearly-changelog>` and :doc:`known issues </docs/yearly-overview/known-issues>` for the season.
+
+.. mimictoc::
+   :maxdepth: 2
```

Comment:
Is there a way to not expand the individual articles? expanding the imaging articles to headings doesn't make sense